styles/styles.css

```diff
@@ -524,6 +524,55 @@ main .section.brands-top>.section-container {
   padding: 0 1rem;
 }
 
+/* section - social */
+main .section.social-media {
+  padding: 4rem 0;
```

Comment:
A section can have multiple variants. I am wondering if permutations of those variants would end up breaking styles such as these. For e.g. if another section comes with its own padding.
 Is there possibility of reusing any of the `social` styles that are already present? /cc @jindaliiita

Reply:
<img width="1706" alt="image" src="https://github.com/hlxsites/sunstar/assets/57093988/101b520d-cf85-4bb6-bd91-082a734081d4">

@sdmcraft  @jindaliiita Actually, in my section, I reuse the social plugin. That's why I didn't create a block for it because I don't want to nest blocks. 

Additionally, I think Piyush created the following blocks for Sunstar.com, unfortunately it's difficult to reuse them except icon images.



<img width="485" alt="image" src="https://github.com/hlxsites/sunstar/assets/57093988/ec68c224-153d-4c3e-a5aa-860f304c5255">

<img width="977" alt="image" src="https://github.com/hlxsites/sunstar/assets/57093988/b99c6272-7c4f-4cd1-9b33-f89c63b01750">